core/src/main/java/org/elasticsearch/common/settings/Settings.java

```diff
@@ -735,14 +735,10 @@ public static void writeSettingsToStream(Settings settings, StreamOutput out) th
         }
     }
 
-    public static Builder builder() {
```

Comment:
Lol this is the least helpful way possible for github to present this diff. I wonder if that is how git presents it?